src/GafferScene/Shader.cpp

```diff
@@ -150,14 +150,14 @@ class Shader::NetworkBuilder
 
 	public :
 
-		NetworkBuilder( const Shader *rootNode )
-			:	m_rootNode( rootNode ), m_handleCount( 0 )
+		NetworkBuilder( const Gaffer::Plug *output )
```

Comment:
Nave I messed up the hashing of the NetworkBuilder given it is now uses a specific plug rather than the Shader node?